.ci_support/linux_64_.yaml

```
@@ -7,4 +7,4 @@ channel_targets:
 docker_image:
 - quay.io/condaforge/linux-anvil-x86_64:alma9
 python_min:
-- '3.11'
+- '3.9'
```

I don't know why this has changed so I'm going to change it back to what I know worked.
```suggestion
- '3.11'
```